src/history/history.service.ts

```diff
@@ -191,7 +188,10 @@ export class HistoryService {
    */
   toHistoryEntryDto(entry: HistoryEntry): HistoryEntryDto {
     return {
-      identifier: entry.note.alias ? entry.note.alias : entry.note.id,
+      identifier:
```

Comment:
Phew, a ternary operator over three lines, I feel a separate function is warranted